opentelemetry-otlp/src/exporter/tonic/mod.rs

```diff
@@ -207,20 +220,107 @@ impl TonicExporterBuilder {
         };
 
         #[cfg(feature = "tls")]
-        let channel = match self.tonic_config.tls_config {
-            Some(tls_config) => endpoint
-                .tls_config(tls_config)
-                .map_err(crate::Error::from)?,
-            None => endpoint,
+        {
+            let insecure = config.insecure.unwrap_or_else(|| {
+                env::var(signal_insecure_var)
+                    .or_else(|_| env::var(OTEL_EXPORTER_OTLP_INSECURE))
```

Comment:
haven't done a full look, but wondering what is the priority order when signal specific ENV variables and this is present?

Reply:
Looking at the code (specifically resolve_tls_config), the priority order seems to be:
`ClientTlsConfig` -> `signal_specific_env` -> `otlp_generic_env`

Reply:
Looking at the code (specifically resolve_tls_config), the priority order seems to be:
`ClientTlsConfig` -> `signal_specific_env` -> `otlp_generic_env`

But need @jvanz to confirm :)